.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: 'circleci/node:latest'
+    steps:
+      - checkout
+      - run:
+          name: install
+          command: npm install
+      - run:
+          name: lint
+          command: npm run lint
+      - run:
+          name: test
+          command: npm run build
+      - run:
+          name: build
+          command: npm run test
+      - run:
+          name: release
+          command: npx semantic-release
```

Comment:
Why use npx if you've installed in devdep?

Reply:
I didnt it's automatically added by the cli. I prefer npx over this, it's more resilient (like if they stop adding this as a running script because with npx it's useless)